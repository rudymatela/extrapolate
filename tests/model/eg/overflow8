*** Failed! Falsifiable (after 2016 tests):
T [] [-1] [-128]

Conditional Generalization:
T xs ((-1):xs) ((-128):xs)  when  0 == sum xs

