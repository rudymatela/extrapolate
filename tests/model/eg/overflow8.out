*** Failed! Falsifiable (after 896 tests):
T [-1] [-128]

Conditional Generalization:
T ((-1):xs) ((-128):xs)  when  0 == sum xs

